### DISCUSSION TITLE
Requirements for listing the project on the assets page

### DISCUSSION THREAD

**Discussion by JhnW** [bevyengine/bevy#4746] [Q&A]
Hi all.
What are the requirements for the project to be listed on https://bevyengine.org/assets/?
Recently I wanted to learn Bevy (and do more in Rust than just syntax quirks) and implemented Snake (https://github.com/JhnW/AnotherRustBevySnakeGame). 
I think that a simple example of Snake could be useful to a few people (on the same principle as Breakout, etc.).

**Comment by mockersf** [comment]
You can submit a PR to https://github.com/bevyengine/bevy-assets and it will be available on https://bevyengine.org/assets/ once merged!

The format of the PR is mostly described in the readme 🙂 

As for requirements, if it uses Bevy you should be good to go 👍
